Python/compile.c

```diff
@@ -45,14 +46,26 @@
         && (c->u->u_ste->ste_type == ModuleBlock))
 
 struct instr {
-    unsigned i_jabs : 1;
-    unsigned i_jrel : 1;
     unsigned char i_opcode;
     int i_oparg;
     struct basicblock_ *i_target; /* target block (if jump instruction) */
     int i_lineno;
 };
 
+static int
+is_relative_jump(struct instr *i)
+{
+    int opcode = i->i_opcode;
+    return (_PyOpcode_RelativeJump[opcode>>5]>>(opcode&31))&1;
```

Comment:
Can you define 5 and 31 as globals with a name (using #define or with static variables). That way this will read a bit better